napari_allencell_annotator/controller/annotator_controller.py

```diff
@@ -145,9 +172,12 @@ def set_curr_img(self, curr_img: Optional[Dict[str, str]] = None):
             # if at the end disable next
             if int(curr_img["Row"]) == self.view.num_images - 1:
```

Comment:
could you clean this if/else up?
I feel like you could do this
```
if int(curr_img["Row"]) == self.view.num_images - 1 or int(curr_img["Row"]) == 0:
self.view.next_btn.setEnabled(False)
else:
True
```

Reply:
I think I need both, if row is len-1 you enable the next button whereas if len  is 0 you enable the previous button so I would need another if statement anyways